sensors/jira_sensor.py

```diff
@@ -41,29 +42,39 @@ def setup(self):
         self._jira_url = self._config['url']
         auth_method = self._config['auth_method']
 
+        options = {'server': self._config['url'],
```

Comment:
Would like to see unit tests for these new options.

Reply:
I have already added these configurations in test fixtures which is being used my tests to run the test.